quickwit/quickwit-ingest/src/ingest_v2/router.rs

```diff
@@ -53,7 +54,7 @@ use crate::LeaderId;
 pub(super) const INGEST_REQUEST_TIMEOUT: Duration = if cfg!(any(test, feature = "testsuite")) {
     Duration::from_millis(10)
 } else {
-    Duration::from_secs(35)
+    Duration::from_secs(8)
```

Comment:
Please update `ingest.md` accordingly. This is also screwing up the timeout values for persist and replicate and the number of retries.

Reply:
ill stick to the previous timeout for the moment then